server/cmwell-ws/app/actions/ActiveInfotonGenerator.scala

```diff
@@ -494,22 +494,13 @@ class ActiveInfotonGenerator @Inject()(
 """
   }
 
-  private[this] val ghdfBreakout =
-    scala.collection.breakOut[Seq[
-      (String,
-       ((String, Color.Color),
-        (String, Color.Color),
-        (String, Color.Color),
-        (String, Color.Color),
-        (String, Color.Color),
-        (String, Color.Color)))
-    ], (String, Set[FieldValue]), Map[String, Set[FieldValue]]]
   private[this] def generateHealthDetailedFields: FieldsOpt = {
     //val (xs, timeStamp) = DashBoardCache.cacheAndGetDetailedHealthData
     val res = getClusterDetailedHealthNew
     Some(
       res.toSeq
         .sortBy(_._1)
+        .view
```

Comment:
Consider to refactor

Reply:
Sort must have all the elements. I couldn't find a way to sort directly on a Map.